xbmc/android/activity/XBMCApp.cpp

```diff
@@ -230,10 +228,6 @@ void CXBMCApp::onDestroyWindow()
     XBMC_DestroyDisplay();
     XBMC_Pause(true);
   }
-
-  if (m_wakeLock)
-    m_wakeLock->release();
```

Comment:
I've moved it to "OnPause", which is the mirror of "OnResume" where we grab the lock.
I prefer to use those as they directly map to the droid activity lifecycle, so it's clear when those happen.